README.md

```diff
@@ -0,0 +1,260 @@
+# blurb
+
+[![PyPI version](https://img.shields.io/pypi/v/blurb.svg?logo=pypi&logoColor=FFE873)](https://pypi.org/project/blurb)
+[![GitHub Actions](https://github.com/python/blurb/actions/workflows/test.yml/badge.svg)](https://github.com/python/blurb/actions)
+[![Codecov](https://codecov.io/gh/python/blurb/branch/main/graph/badge.svg)](https://codecov.io/gh/python/blurb)
+[![Python discussions](https://img.shields.io/badge/Discourse-join_chat-brightgreen.svg)](https://discuss.python.org/)
+
+## Overview
+
+**blurb** is a tool designed to rid CPython core development
+of the scourge of `Misc/NEWS` conflicts.
+
+The core concept: split `Misc/NEWS` into many
+separate files that, when concatenated back together
+in sorted order, reconstitute the original `Misc/NEWS` file.
+After that, `Misc/NEWS` could be deleted from the CPython
+repo and thereafter rendered on demand (e.g. when building
+a release).  When committing a change to CPython, the commit
+process will write out a new file that sorts into the correct place,
+using a filename unlikely to have a merge conflict.
+
+**blurb** is a single command with a number of subcommands.
+It's designed  to be run inside a valid CPython (Git) repo,
+and automatically uses the correct file paths.
+
+You can install **blurb** from PyPI using `pip`.  Alternatively,
+simply add `blurb` to a directory on your path.
+
+**blurb**'s only dependency is Python 3.8+.
+
+
+## Files used by blurb
+
+**blurb** uses a new directory tree called `Misc/NEWS.d`.
+Everything it does is in there, except for possibly
+modifying `Misc/NEWS`.
+
+Under `Misc/NEWS.d` you'll find the following:
+
+* A single file for all news entries per previous revision,
+  named for the exact version number, with the extension `.rst`.
+  Example: `Misc/NEWS.d/3.6.0b2.rst`.
+
+* The `next` directory, which contains subdirectories representing
+  the various `Misc/NEWS` categories.  Inside these subdirectories
+  are more `.rst` files with long, uninteresting, computer-generated
+  names.  Example:
+  `Misc/NEWS.d/next/Library/2017-05-04-12-24-06.gh-issue-25458.Yl4gI2.rst`
+
+
+## blurb subcommands
+
+Like many modern utilities, **blurb** has only one executable
+(called `blurb`), but provides a diverse set of functionality
+through subcommands.  The subcommand is the first argument specified
+on the command-line.
+
+If you're a CPython contributor, you probably don't need to use
+anything except `blurb add`--and you don't even need to specify
```

Comment:
Markdown doesn't convert `--`/`---` to en/em dashes, and the resulting "`blurb add`--and" looks a bit off to me.  We could either replace it with an actual em dash, or at least add spaces (or both).  Pick one:
```suggestion
anything except `blurb add`—and you don't even need to specify
```
```suggestion
anything except `blurb add` — and you don't even need to specify
```
```suggestion
anything except `blurb add` -- and you don't even need to specify
```